src/lib/components/PricingPlan.svelte

```
@@ -0,0 +1,121 @@
+      {#each features as feature}
+        <li class="flex items-center gap-3">
+          {#if isPrimary}
+            <div class="bg-primary/20 rounded-full p-1">
+              <Check class="text-primary h-3 w-3" />
+            </div>
+          {:else}
+            <Check class="h-5 w-5 text-green-500" />
+          {/if}
+          <span>{@html feature}</span>
+        </li>
+      {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)